docs/releases/4.0.txt

```diff
@@ -432,6 +432,9 @@ Miscellaneous
 * The ``object`` argument of undocumented ``ModelAdmin.log_addition()``,
   ``log_change()``, and ``log_deletion()`` methods is renamed to ``obj``.
 
+* :class:`~django.utils.feedgenerator.RssFeed` and `~django.utils.feedgenerator.Atom1Feed` and their subclasses now emit
+  elements with no content as a self-closed tags.
```

Comment:
Wrap at 79 chars.

Reply:
thanks for the feedback. totally missed these two.